src/plugins/memdump/userhook.cpp

```diff
@@ -219,13 +219,8 @@ void memdump::setup_dotnet_hooks(drakvuf_t drakvuf, const char* dll_name, const
         return;
     }
 
-    plugin_target_config_entry_t entry;
-    entry.function_name = "AssemblyNative::LoadImage";
-    entry.dll_name = dll_name;
-    entry.type = HOOK_BY_OFFSET;
-    entry.offset = func_rva;
-    entry.actions = HookActions::empty().set_log().set_stack();
-    this->wanted_hooks.add_hook(std::move(entry));
```

Comment:
This seems to be an unrelated change

Reply:
Right. I decided not to move this change in a separate request. But I can do if you ask.

Reply:
I remove this change from PR